restore-sql-server-transaction-logs/scheduled-upload/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file is not mentioned in the README. What shall I do with it?

Reply:
Removed the need of settings.json file. The variables are defined in the PS script.